Morphology/Regions.cs

```diff
@@ -66,6 +69,10 @@ public string Folder
             Add(new Region("Pose/Head/Mouth/Tongue", true));
             Add(new Region("Pose/Head/Nose", true));
             Add(new Region("Pose/Head/Visemes", true));
+
+            //Apply Green Color for VamStandartRegions to make them Better Visible
+            //Could be moved to ApplyVaMStandardRegions Region Constructors. This is just Lazy
+            this.ToList().ForEach(x => x.DisplayColor = Brushes.Green);
```

Comment:
@Neakas Let's make this a convention in the Region constructor. When a region is instantiated as a standard region, then the brush is gray, otherwise it is light blue.

I want to mimic the VaM styling of custom morphs (blue) vs built-in (gray). Thanks!